front/components/vaults/EditVaultManagedDatasourcesViews.tsx

```diff
@@ -48,8 +49,7 @@ export function EditVaultManagedDataSourcesViews({
   const updateVaultDataSourceViews = async (
     selectedNodes: ManagedDataSourceViewsSelectedNodes
   ) => {
-    let error = null;
-    await Promise.all(
+    const promisesErrors = await Promise.all(
```

Comment:
`nit` Result would have been more elegant.

Reply:
yes but I'm setting it only if it's an error 😅